### PR TITLE
RX65N USB Host Driver Support

### PR DESCRIPTION
**Pull request by anjana-tel**
## Summary
RX65N USB Host Driver Support

## Testing
Tested on RX65N Reference board

### CODE REVIEW

File: arch/renesas/src/rx65n/rx65n_usbhost.h [hunk 1]
```diff
@@ -0,0 +1,263 @@
+/****************************************************************************
+ * arch/renesas/src/rx65n/rx65n/rx6_usbhost.h
+ *
+ *   Copyright (C) 2010-2012, 2014-2017 Gregory Nutt. All rights reserved.
```
Comment (xiaoxiang781216) [inline arch/renesas/src/rx65n/rx65n_usbhost.h:4]: change to apache copyright

**Review by xiaoxiang781216 — COMMENTED** [review]
@anjana-tel could you remove the second patch(Mrge branch...) from PR?

**Comment by anjana-tel** [comment]
> @anjana-tel could you remove the second patch(Mrge branch...) from PR?

Can you suggest how to remove the Patch((Mrge branch...) from PR ?

**Comment by patacongo** [comment]
> 
> 
> > @anjana-tel could you remove the second patch(Mrge branch...) from PR?
> 
> Can you suggest how to remove the Patch((Mrge branch...) from PR ?

Interactive rebase:  git rebase -i HEAD~n
where n is the last n commits including the one you want to remove.

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by xiaoxiang781216** [comment]
LGTM.

**Pull request merged**